IDropApi.cs

```diff
@@ -7,6 +7,6 @@ namespace DropDownloadCore
     // Let us mock out drop http calls.
     public interface IDropApi
     {
-         Task<ISet<VstsFile>> GetVstsManifest(Uri manifestUri, string blobapiversion, string relativeroot);
+         Task<IList<VstsFile>> GetVstsManifest(Uri manifestUri, string blobapiversion, string relativeroot);
```

Comment:
Theoretically the vstsfile's should be distinct hence the set.

Reply:
But the API returns a json list `[]` and not a dictionary`{}`, so I think it's more natural to use a list than to copy all of the files to a hash set for theoretical purposes. Most of the code is just enumerating through it, which feels better with a list to me also.